src/App/App.js

```diff
@@ -3,39 +3,36 @@ import "../styles.css";
 import { useState } from 'react';
 
 const generateCardList = () => {
-  const cardlist = [
-    "100000010",
-    "100000011",
-    "100000012",
-    "100000013",
-    "100000014",
-    "100000015",
-    "100000020",
-    "100000021",
-    "100000010",
-    "100000011",
-    "100000012",
-    "100000013",
-    "100000014",
-    "100000015",
-    "100000020",
-    "100000021",
-  ];
+  const cardlist = [];
+
+  while (cardlist.length < 15) {
+
+    var max = 95;
+    var min = 79;
+
+    const selectNum = Math.floor(Math.random() * (max - min)) + min;
+
+    if (!(cardlist.includes(selectNum.toString()))) {
+
+      cardlist.push(selectNum.toString());
+      cardlist.push(selectNum.toString());
+    }
```

Comment:
No caso da solução da soma desse número com `100000000`, não precisa converter pra string, daí o código já fica mais simples.

```suggestion
    if (!cardlist.includes(selectNum)) {
      cardlist.push(selectNum);
      cardlist.push(selectNum);
    }
```